web/server/parser/packageParser.go

```diff
@@ -139,19 +139,17 @@ func (self *outputParser) recordTestingOutcome(outcome string) {
 	self.result.Elapsed = parseDurationInSeconds(fields[2], 3)
 }
 
-var coverageStatementRE = regexp.MustCompile(`coverage: (\d+\.\d)% of statements`)
+var coverageStatementRE = regexp.MustCompile(`coverage: (\d+\.\d)%%? of statements`)
```

Comment:
This will be due to a Go version change - can you bump go.mod to match?

Reply:
this is actually broken at the current version in go.mod (1.18). I do want to upgrade to a supported go version (1.21 now is the earliest such option) and upgrade external deps, but I figured I'd do that in a separate PR. This effort to fix the broken tests (and there's still another, bigger one) has made me want to address a bunch of things

```
% go1.18 test ./web/server/parser/...
--- FAIL: TestParsePackage_OldSchoolWithSuccessOutput_ReturnsCompletePackageResult (0.00s)
panic: recordCoverageSummary [recovered]
	panic: recordCoverageSummary
```

Reply:
ACK